features/enable_fips_pro.feature

```diff
@@ -140,12 +142,13 @@ Feature: FIPS enablement in PRO cloud based machines
             fips-updates  +yes +disabled +FIPS compliant crypto packages with stable security updates
             """
         When I run `pro enable <fips-service> --assume-yes` with sudo
-        Then stdout matches regexp:
+        Then stdout contains substr:
```

Comment:
Unfortunately the step currently only works if you spell out `substring`
```suggestion
        Then stdout contains substring:
```

Reply:
Fixed